### ISSUE TITLE
Export graph with description

### ISSUE THREAD

**Issue by bzhpwr** [RealOpInsightLabs/realopinsight-workstation#75]
Hi @RealOpInsightLabs,

My boss ask me if rois could export graph with description from web interface.

I know actually it's not possible but can you implement that in next release ?

**Comment by rchakode** [comment]
Hi,
Yes, this sounds possible.  Perhaps not for the next release (which will focus on memory & performance improvements), but likely for the release after.
Thanks.

**Comment by bzhpwr** [comment]
Hi @rchakode,

For me, it's not a priority so I'm agree with you.
I have memory leak (from realopinsight.fcgi) too so it's more important for me. 
Actually I restart realopinsight-reportd and apache, then I kill all realopinsight.fcgi,  every 3 hours.

**Comment by rchakode** [comment]
Yes,  we have also found that, and think that it's really inconvenient.
 So we work to fix that.
